docs/contribute/cumulative-docs/example-scenarios.md

```diff
@@ -0,0 +1,537 @@
+
+**When to use inline `applies_to`**:
+
+* Workflow steps that vary between situations can be easily isolated.
+* Each step that varies, only varies between 3 or fewer situations (deployment types, versions, etc).
+* There are no more than 3 steps that need to be split into multiple lines with `applies_to` badges.
+
+**Best practices**:
+
+* Follow the [best practices for ordered and unordered lists](/contribute/cumulative-docs/badge-placement.md#ordered-and-unordered-lists)
+  including the order of items and the placement of labels.
+
+**Example**: Only one item in an ordered list varies between Serverless and stateful.
+
+::::{image} ./images/workflow-inline.png
+:screenshot:
+::::
+
+### Solution B: Use tabs [workflow-tabs]
+
+Tabs are minimally disruptive in many situations.
+
+**When to use tabs**:
+
+* Using [inline `applies_to` badges](#workflow-inline) isn't appropriate.
+* All the tabs fit horizontally on a single row on a typical laptop screen.
+  This is usually around a maximum of four tabs.
+* The tab with the most content fits vertically on a typical laptop screen.
+  This is usually around of 20 lines.
+
+**Best practices**:
+
+* Try to minimize the number of tabs where possible. Try to work around small differences by
+  rewording or adding context in prose or in `note` style admonitions.
+* Try not to include information surrounding a procedure in the tabs.
+  Make the tab content as small as possible apart from the procedure itself.
+* Consider breaking up procedures into sets of procedures if only one section differs between contexts.
+
+**Example**: On the [Enable audit logging](https://www.elastic.co/docs/deploy-manage/security/logging-configuration/enabling-audit-logs#enable-audit-logging-procedure) page, we use tabs to show separate ordered lists outlining the workflow for each deployment type.
+
+::::{image} ./images/example-workflow-tabs.png
+:screenshot:
+::::
+
+### Solution C: Use sibling pages [workflow-sibling-pages]
+
+Sibling pages are a last resort when no other solutions are appropriate.
+
+**When to use sibling pages**:
+
+* Neither [inline `applies_to` badges](#workflow-inline) or [tabs](#workflow-tabs) are appropriate.
+* The workflow has significant differences across multiple procedures.
+* There are chained procedures where not all of the procedures are needed for all contexts
+  or where the flow across procedures is muddied when versioning context is added.
+* The workflow exists in a very complex page that is already heavily using tabs and other tools we use for versioning differences.
+  This makes it difficult to add another “layer” of content.
+* Product lifecycle state changes like when technical preview or beta transitions to GA.
+
+**Best practices**:
+
+* When using this strategy for product lifecycle changes: hide the earlier page and cross-link between the two so the more stable GA doc is more prominent, but the earlier version is still available to users who need it.
```

Comment:
Are we sure this is how we want to do this?

An option could be to specify the version in the page title, and have the several pages live next to each other, or older versions as subpages of the latest.

I'm just really not sure about hiding the pages.

Reply:
I'd say cut it for now